packages/core/src/codewhisperer/models/constants.ts

```diff
@@ -477,6 +481,8 @@ export const failedStepMessage = 'The step failed, fetching additional details..
 
 export const jobCompletedMessage = 'The transformation completed.'
 
+export const noChangesMadeMessage = "I didn't make any changes for this transformation."
```

Comment:
nit: Will the didn't appear or do we need to deal with apostrophe special character? I notice sometimes we have the backslash and sometimes we don't, so just want to make sure